podcasts/PlayerContainerViewController.swift

```diff
@@ -248,7 +248,7 @@ class PlayerContainerViewController: SimpleNotificationsViewController, PlayerTa
     private func adjustHeaderConstraintIfNeeded() {
         guard let window = view.window else { return }
 
-        let requiredHeight = 45 + UIUtil.statusBarHeight(in: window)
+        let requiredHeight = 50 + UIUtil.statusBarHeight(in: window)
```

Comment:
I tried to used the SafeArea object instead of this, but there was complications with the full screen player animations so I opted for this simpler approach.